### PR TITLE
Chanakya/vault dashboard styling

### PR DESCRIPTION
**Pull request by Chanakya888**
Please ignore the node-sass error, I will be bringing it back with my last commit (fixed now)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "i18next": "^19.8.4",
     "i18next-browser-languagedetector": "^6.0.1",
     "i18next-http-backend": "^1.0.21",
-    "node-sass": "^4.14.1",
+     "node-sass": "^4.14.1",
```
Comment (nud3l) [inline package.json:37]: remove the extra space please

File: src/assets/locales/en/translation.json [hunk 2]
```diff
@@ -291,8 +291,6 @@
       "withdraw_collateral": "Withdraw Collateral",
       "add_collateral": "Add Collateral",
       "vault_dashboard": "Vault Dashboard",
-      "locked_collateral": "Locked collateral",
```
Comment (nud3l) [inline src/assets/locales/en/translation.json:294]: Any reasons those are removed? At least on my end, this breaks the main dashboard

File: src/components/CardList/index.tsx [hunk 3]
```diff
@@ -13,22 +13,16 @@ const Card = ({
 }: Props & React.ComponentPropsWithRef<'li'>): JSX.Element => (
   <li
     // TODO: hardcoded
-    style={{ minHeight: 128 }}
```
Comment (nud3l) [inline src/components/CardList/index.tsx:16]: Cards are overlapping in my default resolution and this is also breaking when you change the resolution:

https://user-images.githubusercontent.com/14966470/119681454-0dfff600-be7d-11eb-9433-fa0a3c43288c.mp4

File: src/pages/Challenges/index.tsx [hunk 4]
```diff
@@ -144,22 +144,24 @@ function Challenges(): JSX.Element {
       )}>
       <div className='w-3/4'>
         <PageTitle mainTitle={t('leaderboard.challenges_title')} />
-        <CardList>
+        <CardList className='grid-cols-3 gap-5 2xl:grid-cols-5 2xl:gap-6'>
```
Comment (nud3l) [inline src/pages/Challenges/index.tsx:147]: No extra classes here -> should be added to the general card list to keep one common design

File: src/pages/Feedback/index.tsx [hunk 5]
```diff
@@ -49,27 +50,36 @@ const FEEDBACK_ITEMS = [
...
-      </CardList>
+    <MainContainer
+      className={clsx(
+        'flex',
+        'justify-center',
+        'fade-in-animation'
+      )}>
+      <div className='w-3/4'>
+        <PageTitle mainTitle='Feedback' />
+        <CardList className='grid-cols-3 gap-4 2xl:grid-cols-5'>
```
Comment (nud3l) [inline src/pages/Feedback/index.tsx:61]: Same here: no classes in the pages. Should be done directly on the `CardList` component

File: src/pages/dashboard/components/collateralization.tsx [hunk 6]
```diff
@@ -63,7 +63,7 @@ export default function Collateralization({ linkButton }: CollateralizationProps
         <div className='values-container'>
           {!failed && (
             <>
-              <h1 className='text-interlayDodgerBlue'>{t('dashboard.vault.collateralization')}</h1>
+              <h1 className='text-'>{t('dashboard.vault.collateralization')}</h1>
```
Comment (nud3l) [inline src/pages/dashboard/components/collateralization.tsx:66]: what is the `text-` class?

File: src/assets/locales/en/translation.json (hunk 2 above)
Comment (Chanakya888) [reply src/assets/locales/en/translation.json:294]: My confusion sorry, there are two vaults (identical in name), one inside dashboard and one outside it. Will fix this

File: src/components/CardList/index.tsx (hunk 3 above)
Comment (Chanakya888) [reply src/components/CardList/index.tsx:16]: Whoops, I added a "w-72" class to test the layout and forgot to remove it. Removed it now.

File: src/pages/Challenges/index.tsx (hunk 4 above)
Comment (Chanakya888) [reply src/pages/Challenges/index.tsx]: I don't think having a common design for all the layouts is a good idea, my thoughts were:

1. In the challenges page, the idea of having 3 in row 1 and 2 in row 2 for laptops is that the 1,2,3 challenges are already done, 4th and 5th challenge are the new ones, having them in a new line stands them out apart from the rest of the completed challenges (exception: ultrawide screens wherein I used 1*5 layout)
2. In the current layout, there are certain screen sizes where there is only element left out in a row, which isn't ideal [ref](https://indesignskills.com/tutorials/widows-and-orphans/)
3. I wasn't satisfied with Vault Dashboard design yesterday, so I swapped the positions, prioritised fees and collateralization which are the most useful things to a user (will remain front and center in every screen size)

File: src/pages/dashboard/components/collateralization.tsx (hunk 6 above)
Comment (Chanakya888) [reply src/pages/dashboard/components/collateralization.tsx:66]: That's the text color (blue). Anton gave a quirky name lol

File: src/pages/Feedback/index.tsx (hunk 5 above)
Comment (Chanakya888) [reply src/pages/Feedback/index.tsx]: Tbh, this classname is a prop, it puts the prop in the `CardList` component itself

File: src/pages/dashboard/components/collateralization.tsx (hunk 6 above)
Comment (nud3l) [reply src/pages/dashboard/components/collateralization.tsx:66]: but you deleted it ;)

Comment (Chanakya888) [reply src/pages/dashboard/components/collateralization.tsx]: ah, hate myself. Fixed it

**Pull request closed without merging**